docs/deploy-to-github.md

```diff
@@ -41,6 +41,11 @@ GitHub Pages allows you to have a "GitHub user page" that acts as a profile/main
   ```js
   siteUrl: 'https://www.yourname.com',
   ```
+  * In addition to this, you will want to create a `CNAME` file inside `/static` to avoid a known issue in GitHub Pages that results in the deployment [incorrectly overwriting the custom domain](https://github.com/tschaub/gh-pages/issues/213) setting within GitHub user settings.
+    ```js
```

Comment:
Could you update this to remove the `js` language tag? I'm assuming it's just a plain text file.